student.rb

```diff
@@ -0,0 +1,12 @@
+require './Person'
+
+class Student < Person
+  def initialize(classroom)
+    super(age, name, parent_permission: true)
```

Comment:
- [ ] Kindly add `age`, `name`, and `parent_permission` variables to `initialize` method parameters beside classroom because this is the right way to inherit these variables.

[Hint]: the initialize method will be something like that:
```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
end
```

Reply:
Well resolved 👍🏽